src/ECER.Clients.RegistryPortal/ECER.Clients.RegistryPortal.Server/Applications/ApplicationsEndpoints.cs

```diff
@@ -124,6 +126,10 @@ public record Transcript()
   [Required]
   public DateTime EndDate { get; set; }
 }
+public record WorkExperienceReference(string? Id, [Required] string? FirstName, [Required] string? LastName, [Required] string? EmailAddress)
```

Comment:
@farzadnadiri @ytqsl do we also want to validate field length here? 

`[StringLength(100, ErrorMessage = "...")]`

Or is that to be handled exclusively on the client? As is now we will 500 on the server when attempting to save to dynamics with greater than acceptable field length.

Reply:
I remember when we talked about backend validation. we just wanted to cover required fields but if we want we can add that as well.

Reply:
Sounds good